### PR TITLE
Std hotfix request

### PR DESCRIPTION
**Pull request by gcezar**
Added verify=False for all requests and fixed some minor typos

### CODE REVIEW

File: portal/app/farm_updater/std_api.py [hunk 1]
```diff
@@ -8,7 +8,8 @@ class StdApiInterface:
     """
 
     LOGIN_ENDPOINT = '/api/login'
-    DEVICE_ENDPOINT = '/api/device'
+    DEVICES_ENDPOINT = '/api/device'
+    DEVICE_ENDPOINT = '/api/device/'
```
Comment (jongoncalves) [inline portal/app/farm_updater/std_api.py]: These are the exact same. Why use two?
Just format the string if you need to give it a device id - no reason to maintain both

File: portal/app/farm_updater/std_api.py [hunk 2]
```diff
@@ -17,34 +17,37 @@ def __init__(self, url=settings.SUN_TECH_DRIVE_URL, username=None, password=None
 
     def login(self):
         session = requests.Session()
-        session.post(self.url + self.LOGIN_ENDPOINT, data=self.credentials)
+        try:
+            session.post(self.url + self.LOGIN_ENDPOINT, data=self.credentials, verify=False)
+        except requests.exceptions.RequestException as exc:
```
Comment (jongoncalves) [inline portal/app/farm_updater/std_api.py]: Why did you add this try except? All you do is print the exception but then you're still using the session?

Comment (gcezar) [reply portal/app/farm_updater/std_api.py]: forgot to delete that...
